controllers/roleController.js

```diff
@@ -0,0 +1,13 @@
+const Role = require("../models/roles");
+const sequelize = require("../database/db");
+module.exports = {
+  addRole: async (req, res) => {
+    const { name, description } = req.body;
+    const createUser = await Role.create({
```

Comment:
createUser should be createRole. You are not going to create user, it will create role